src/jttt/Core/AlphaBetaStrategy.java

```diff
@@ -0,0 +1,112 @@
+package jttt.Core;
+
+import java.util.HashMap;
+
+import static java.lang.Math.max;
+import static java.lang.Math.min;
+
+public class AlphaBetaStrategy implements AIMoveStrategy {
+    private static int DISPLAY_POSITION_OFFSET = 1;
+    private static final int SCORE_FOR_DRAW = 0;
+    private static Integer DEPTH = 0;
+    private static int INITIAL_SCORE = 1000000;
+    private static int INITIAL_ALPHA = -10000000;
+    private static int INITIAL_BETA = 10000000;
+    private static String MOVE_KEY = "Move";
+    private static String SCORE_KEY = "Score";
+    private Counter counter;
```

Comment:
`Counter` sounds like it would "count" something... maybe `Mark` or `Symbol` is more intention revealing?